mobile/src/screens/survey/survey.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import { Link, SignBackground, Text } from '#libs/components/components';
+import { RootScreenName } from '#libs/enums/enums';
+
+import { PreferencesStep } from './components/components';
+import { styles } from './styles';
+
+const Survey: React.FC = () => {
+  return (
+    <React.Fragment>
+      <SignBackground>
+        <Text style={styles.labelText}>
+          <Link label="Sign In" to={`/${RootScreenName.SIGN_IN}`} />
```

Comment:
Do RootScreenName.SIGN_IN starts with '/'?

Reply:
@v0ldemar01 it was written in the project starter like this (in created component sign up). and if i change it I get an error: ERROR Error: The path must start with '/' (Sign In)., so I left everything as it was

Reply:
Good explanation)